### PR TITLE
Move Examples to Use Non Deprecated Classes

### PR DESCRIPTION
**Pull request by lilleswing**
I went through the examples and replaced usages of deprecated models with their newer counterparts.  I smoke tested the results w/ fewer epochs for similar performance.

### CODE REVIEW

File: deepchem/models/tensorgraph/models/graph_models.py [hunk 1]
```diff
@@ -338,6 +338,18 @@ def default_generator(self,
 
         yield feed_dict
 
+  def predict(self, dataset, transformers=[], outputs=None):
```
Comment (lilleswing) [inline deepchem/models/tensorgraph/models/graph_models.py]: added undo-ing transformers to DTNN

File: examples/bace/bace_dnn.py [hunk 2]
```diff
@@ -1,101 +0,0 @@
...
-from bace_datasets import load_bace
-from deepchem.utils.save import load_from_disk
-from deepchem.splits import SpecifiedSplitter
-from deepchem.data import Dataset
-from deepchem.hyper import HyperparamOpt
-from deepchem import metrics
-from deepchem.metrics import Metric
-from deepchem.utils.evaluate import Evaluator
-from deepchem.models.keras_models.fcnet import MultiTaskDNN
-from deepchem.models.keras_models import KerasModel
```
Comment (lilleswing) [inline examples/bace/bace_dnn.py]: these are gone -- I would have to go through the git time machine to find them if we really wanted them.

File: examples/qm7/qm7_DTNN.py [hunk 3]
```diff
@@ -1,63 +0,0 @@
-"""
```
Comment (lilleswing) [inline examples/qm7/qm7_DTNN.py]: tensorgraph version already exists.

File: deepchem/models/tensorgraph/models/graph_models.py [hunk 4]
```diff
@@ -31,21 +31,21 @@ def __init__(self,
                mode="classification",
                **kwargs):
     """
-        Parameters
```
Comment (rbharath) [inline deepchem/models/tensorgraph/models/graph_models.py]: The spacing is getting worse. Yapf hates our beautiful docstrings :-)

File: deepchem/models/tensorgraph/models/graph_models.py (hunk 1 above)
Comment (rbharath) [reply deepchem/models/tensorgraph/models/graph_models.py]: Nice!

File: examples/bace/bace_dnn.py (hunk 2 above)
Comment (rbharath) [reply examples/bace/bace_dnn.py]: No worries. It's not worth investing effort into deprecated models

File: examples/delaney/delaney_DAG.py [hunk 5]
```diff
@@ -1,69 +0,0 @@
...
-valid_dataset.reshard(512)
-valid_dataset = transformer.transform(valid_dataset)
-test_dataset.reshard(512)
-test_dataset = transformer.transform(test_dataset)
-# Do setup required for tf/keras models
-# Number of features on conv-mols
-n_feat = 75
-# Batch size of models
-batch_size = 64
-graph = dc.nn.SequentialDAGGraph(n_atom_feat=n_feat, max_atoms=max_atoms)
```
Comment (rbharath) [inline examples/delaney/delaney_DAG.py]: Do we have a TensorGraph version of `SequentialDAGGraph` available? If not, we should add it on to #963.

File: examples/delaney/delaney_weave.py [hunk 6]
```diff
@@ -1,63 +0,0 @@
...
-graph = dc.nn.SequentialWeaveGraph(
-    max_atoms=max_atoms, n_atom_feat=n_atom_feat, n_pair_feat=n_pair_feat)
-
-graph.add(dc.nn.WeaveLayer(max_atoms, 75, 14))
-#graph.add(dc.nn.WeaveLayer(max_atoms, 50, 50))
-graph.add(dc.nn.WeaveConcat(batch_size, n_output=n_feat))
-graph.add(dc.nn.BatchNormalization(epsilon=1e-5, mode=1))
-graph.add(dc.nn.WeaveGather(batch_size, n_input=n_feat, gaussian_expand=True))
-
-model = dc.models.MultitaskGraphRegressor(
```
Comment (rbharath) [inline examples/delaney/delaney_weave.py]: I think there's a new Weave Tensorgraph version I believe.

File: deepchem/models/tensorgraph/models/graph_models.py (hunk 4 above)
Comment (lilleswing) [reply deepchem/models/tensorgraph/models/graph_models.py]: maybe update yapf again?  Also one can fiddle with it I believe if you prefer certain things.

File: examples/delaney/delaney_weave.py (hunk 6 above)
Comment (lilleswing) [reply examples/delaney/delaney_weave.py]: yeah it already exists in this folder so I didn't replace I just left the new one.

**Comment by rbharath** [comment]
Going to go ahead and merge this in to clear the way for #976.

LGTM

**Pull request merged**